src/core/sys/freebsd/netinet/in_.d

```diff
@@ -0,0 +1,581 @@
+
+    enum IP_FLOWID          = 90;
+    enum IP_FLOWTYPE        = 91;
+    enum IP_RSSBUCKETID     = 92;
+    enum IP_RECVFLOWID      = 93;
+    enum IP_RECVRSSBUCKETID = 94;
+
+    enum IP_DEFAULT_MULTICAST_TTL  = 1;
+    enum IP_DEFAULT_MULTICAST_LOOP = 1;
+
+    enum IP_MIN_MEMBERSHIPS   = 31;
+    enum IP_MAX_MEMBERSHIPS   = 4095;
+
+    enum IP_MAX_GROUP_SRC_FILTER = 512;
+    enum IP_MAX_SOCK_SRC_FILTER  = 128;
+
+    struct ip_mreq
+    {
+        in_addr imr_multiaddr;
+        in_addr imr_interface;
+    };
+
+    struct ip_mreqn
+    {
+        in_addr imr_multiaddr;
+        in_addr imr_address;
+        int     imr_ifindex;
+    };
+
+    struct ip_mreq_source
+    {
+        in_addr imr_multiaddr;
+        in_addr imr_sourceaddr;
+        in_addr imr_interface;
+    };
+
+    struct group_req
+    {
+        uint gr_interface;
+        sockaddr_storage gr_group;
+    };
+
+    struct group_source_req
+    {
+        uint gsr_interface;
+        sockaddr_storage gsr_group;
+        sockaddr_storage gsr_source;
+    };
+
+    int setipv4sourcefilter(int, in_addr, in_addr, uint, uint, in_addr*);
+    int getipv4sourcefilter(int, in_addr, in_addr, uint*, uint*, in_addr*);
+    int setsourcefilter(int, uint, sockaddr*, socklen_t, uint, uint, sockaddr_storage*);
+    int getsourcefilter(int, uint, sockaddr*, socklen_t, uint*, uint*, sockaddr_storage*);
+
+    enum MCAST_UNDEFINED = 0;
+    enum MCAST_INCLUDE   = 1;
+    enum MCAST_EXCLUDE   = 2;
+
+    enum IP_PORTRANGE_DEFAULT = 0;
+    enum IP_PORTRANGE_HIGH    = 1;
+    enum IP_PORTRANGE_LOW     = 2;
+
+    enum IPCTL_FORWARDING        = 1;
+    enum IPCTL_SENDREDIRECTS     = 2;
+    enum IPCTL_DEFTTL            = 3;
+    enum IPCTL_DEFMTU            = 4;
+    enum IPCTL_SOURCEROUTE       = 8;
+    enum IPCTL_DIRECTEDBROADCAST = 9;
+    enum IPCTL_INTRQMAXLEN       = 10;
+    enum IPCTL_INTRQDROPS        = 11;
+    enum IPCTL_STATS             = 12;
+    enum IPCTL_ACCEPTSOURCEROUTE = 13;
+    enum IPCTL_FASTFORWARDING    = 14;
+    enum IPCTL_GIF_TTL           = 16;
+    enum IPCTL_INTRDQMAXLEN      = 17;
+    enum IPCTL_INTRDQDROPS       = 18;
+}
+
+// =============================================================================
+// What follows is from netinet6/in6.h, but since netinet6/in6.h specifically
+// says that it should only be #included by #including netinet/in.h, it makes
+// more sense to put them in here so that the way they're imported corresponds
+// with the correct way of #including them in C/C++.
+// =============================================================================
+
+// The #if was around the #include of in6.h at the end of in.h.
+static if (__POSIX_VISIBLE)
+{
+
+    static if (__BSD_VISIBLE)
+    {
+        enum IPV6PORT_RESERVED    = 1024;
+        enum IPV6PORT_ANONMIN     = 49152;
+        enum IPV6PORT_ANONMAX     = 65535;
+        enum IPV6PORT_RESERVEDMIN = 600;
+        enum IPV6PORT_RESERVEDMAX = IPV6PORT_RESERVED - 1;
+
+// TODO
```

Comment:
I was just curious if you left this for latter and then forgot (which happens to me on a regular basis). No, I don't think there's need to add this note - after the review is finished, I don't think the note would carry any information.